src/dbnode/storage/index/results.go

```diff
@@ -84,25 +84,14 @@ func (r *results) Add(d doc.Document) (added bool, size int, err error) {
 func (r *results) tags(fields doc.Fields) ident.Tags {
 	tags := r.idPool.Tags()
 	for _, f := range fields {
-		tags.Append(ident.Tag{
-			Name:  r.copyBytes(f.Name),
-			Value: r.copyBytes(f.Value),
-		})
+		tags.Append(r.idPool.CloneTag(ident.Tag{
```

Comment:
is this just a cleanup?

Reply:
Yup, this wasn't being pooled before, now it is (it always could have been).